packages/components/popover/popover.component.ts

```diff
@@ -165,6 +165,9 @@ export class KbqPopoverTrigger extends KbqPopUpTrigger<KbqPopoverComponent> impl
     /** prevents closure by any event */
     @Input({ alias: 'kbqPopoverPreventClose', transform: booleanAttribute }) override preventClose: boolean = false;
 
+    /** disables padding for all popover elements (header, content and footer) */
+    @Input({ transform: booleanAttribute }) noPaddings = false;
```

Comment:
Может так и назвать, как в описании?

`!noPaddings` тяжелее читается, лично для меня

```suggestion
    @Input({ transform: booleanAttribute }) disablePaddings = false;
```

Reply:
оба варианта конечно так себе... Сделал `defaultPaddings`